lib/fs/basicfs_unix.go

```diff
@@ -14,7 +14,7 @@ func (BasicFilesystem) SymlinksSupported() bool {
 	return true
 }
 
-func (f *BasicFilesystem) CreateSymlink(target, name string) error {
+func (f *BasicFilesystem) CreateSymlink(name, target string) error {
```

Comment:
So I am not sure, if this is right, as this swaps arguments to all other things.

Reply:
at least some tests failing

Reply:
Target (as in what the symlink is pointing to) must be first to do it the same way as `os`.

Reply:
Sorry about that, my bad (pushed and created a PR which wasn't the latest). Updated!